aten/src/ATen/native/cpu/SoftMaxKernel.cpp

```diff
@@ -224,25 +224,26 @@ struct vec_host_softmax_backward_lastdim {
 };
 
 static void softmax_lastdim_kernel_impl(Tensor& result, const Tensor& self) {
-  AT_DISPATCH_FLOATING_TYPES(self.scalar_type(), "softmax_lastdim_kernel_impl", [&] {
-    vec_host_softmax_lastdim<scalar_t, false>::apply(result, self);
+  AT_DISPATCH_FLOATING_TYPES_AND(kBFloat16,
```

Comment:
We have some softmax testing in test_torch, but it looks like we'd need to update the softmax tests in test_nn.py to test this.